Sources/MapboxMaps/Foundation/Camera/FlyToCameraAnimator.swift

```diff
@@ -45,7 +45,7 @@ public class FlyToCameraAnimator: NSObject, CameraAnimator, CameraAnimatorInterf
     }
 
     public func stopAnimation() {
-        state = .stopped
+        state = .inactive
```

Comment:
Note this behavior change in the changelog;

Create a ticket about divorcing our public animator interfaces from the UIKit types